goteoapi/projects/models.py

```diff
@@ -386,6 +397,7 @@ def get_filters(self, **kwargs):
 
         if 'node' in kwargs and kwargs['node'] is not None:
             filters.append(self.node_id.in_(as_list(kwargs['node'])))
+            filters.append(NodeProject.node_id.in_(as_list(kwargs['node'])))
```

Comment:
Is this change going to make any difference?
if we filter in the previous line by the node_id of the project, any other filter will useles, didn't it? Should we need a `OR` here? or remove the previous line?

Reply:
Yes, you are right, we could get rid of the first filter, but for compatibility i think we should leave it there (project.node is not yet deprecated). I will add now the _or to this filter